queries.sql

```diff
@@ -67,3 +67,25 @@ SELECT * FROM animals as Animals JOIN species as Species ON Species.id = Animals
 SELECT * FROM animals as Animals JOIN owners as Owners ON Owners.id = Animals.owner_id WHERE Animals.escape_attempts= 0 AND Owners.full_name= 'Dean Winchester';
 SELECT Animals.owner_id, Owners.full_name FROM animals as Animals INNER JOIN owners as Owners ON Animals.owner_id= Owners.id GROUP BY Animals.owner_id, Owners.full_name
 ORDER BY COUNT(*) DESC LIMIT 1;
+
+/*Project 4 */
+
+
+SELECT vets.name, animals.name, visit_date FROM visits INNER JOIN animals ON animals.id = visits.animals_id INNER JOIN vets ON vets.id = vets_id WHERE visit_date = (SELECT MAX(visit_date) FROM visits where vets_id= 1);
+
+
+SELECT COUNT(animals_id) FROM visits WHERE vets_id= 3;
```

Comment:
- [ ] Good job writing the queries but the query is for `how many DIFFERENT animals did Stephanie Mendez see?`. The keyword here is DIFFERENT.
Kindly use a join to while writing this query.